### PR TITLE
trashy: add page

### PR DESCRIPTION
**Pull request by nath1as**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

File: pages/linux/trashy.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# trashy
+
+> Trashy is a simple, fast, and featureful alternative to rm and trash-cli written in rust.
```
Comment (navarroaxel) [inline pages/linux/trashy.md:3]: We prefer an objective description here
```suggestion
> An alternative to `rm` and `trash-cli` written in Rust.
```

File: pages/linux/trashy.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+> More information: <https://github.com/oberblastmeister/trashy>.
+
+- Trash several files:
+
+`trash {{path/to/file}}`
+
+- Trash several files:
+
+`trash {{path/to/file1}} {{path/to/file2}} {{path/to/file3}}`
```
Comment (navarroaxel) [inline pages/linux/trashy.md:12]: We use this standard
```suggestion
`trash {{path/to/file1 path/to/file2 ...}}`
```

File: pages/linux/trashy.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# trashy
+
+> Trashy is a simple, fast, and featureful alternative to rm and trash-cli written in rust.
+> More information: <https://github.com/oberblastmeister/trashy>.
+
+- Trash several files:
```
Comment (navarroaxel) [inline pages/linux/trashy.md:6]: I'm not sure about use `trash` as a verb
```suggestion
- Move a specific file to the trash:
```

File: pages/linux/trashy.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+
+`trash {{path/to/file}}`
+
+- Trash several files:
```
Comment (navarroaxel) [inline pages/linux/trashy.md:10]:
```suggestion
- Move specific files to the trash:
```

File: pages/linux/trashy.md [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+
+`trash {{path/to/file1}} {{path/to/file2}} {{path/to/file3}}`
+
+- List items in the trash:
+
+`trash list`
+
+- Restore file from trash:
```
Comment (navarroaxel) [inline pages/linux/trashy.md:18]:
```suggestion
- Restore a specific file from the trash:
```

File: pages/linux/trashy.md [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+
+- Empty file from trash:
+
+`trash empty {{file}}`
+
+- Restore all files:
+
+`trash restore --all`
+
+- Empty all files:
```
Comment (navarroaxel) [inline pages/linux/trashy.md:30]:
```suggestion
- Remove all files from the trash:
```

File: pages/linux/trashy.md [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+
+- Restore file from trash:
+
+`trash restore {{file}}`
+
+- Empty file from trash:
+
+`trash empty {{file}}`
+
+- Restore all files:
```
Comment (navarroaxel) [inline pages/linux/trashy.md:26]:
```suggestion
- Restore all files from the trash:
```

File: pages/linux/trashy.md [hunk 8]
```diff
@@ -0,0 +1,32 @@
...
+
+- List items in the trash:
+
+`trash list`
+
+- Restore file from trash:
+
+`trash restore {{file}}`
+
+- Empty file from trash:
```
Comment (navarroaxel) [inline pages/linux/trashy.md:22]:
```suggestion
- Remove a specific file from the trash:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**